source/guides/getting-started.md

```diff
@@ -159,6 +159,16 @@ Note that, although Hanami is ready for a Behavior Driven Development workflow o
 We'll go with [Minitest](https://github.com/seattlerb/minitest) here (which is the default), but we can use [RSpec](http://rspec.info) by creating the project with `--test=rspec` option.
 Hanami will then generate helpers and stub files for it.
 
+Nonetheless we'll have to setup the test database to be able to run the tests.
+First we have to define our database url in `.env.test`.
```

Comment:
@Bounga Can we change this into a _notice_ block? Something like "Please check `.env.test` in case you need to tweak the database URL". Thanks 😄